package.json

```diff
@@ -2,9 +2,10 @@
   "name": "bitbox02-api",
   "version": "0.8.1",
   "description": "BitBox02 JavaScript library – To enable communication from the browser to the BitBox02 hardware wallet",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "prepublish": "babel src --out-dir lib --copy-files --ignore src/bitbox02-api-go.js"
   },
```

Comment:
could you add "files" field to list the files that are intended to be published to npm

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files